package.json

```diff
@@ -7,15 +7,14 @@
     "server": "NODE_PATH=./ ts-node --project server/tsconfig.json server/server.ts",
     "start": "NODE_ENV=production NODE_PATH=\".dist/\" node .dist/server/server.js -p $PORT",
     "test": "jest --watch",
-    "lint": "yarn lint:js && yarn lint:css:in:js && yarn lint:css:in:ts && yarn lint:types && yarn lint:fix",
-    "lint:js": "eslint --ext .js,.jsx,.ts,.tsx ./",
+    "lint": "yarn lint:js && yarn lint:css:in:js && yarn lint:css:in:ts && yarn lint:types",
+    "lint:js": "eslint --fix --ext .js,.jsx,.ts,.tsx ./",
```

Comment:
lint: js is used in ci. We shouldn't change the code in ci because it might lead to some hidden problems.